src/Propel/Generator/Model/Database.php

```diff
@@ -528,10 +539,16 @@ public function addTables(array $tables)
         }
     }
 
+    /**
+     * Removes table from this database.
+     *
+     * @param  Table|array $table
```

Comment:
array?

Reply:
```suggestion
     * @param  Table $table
```

It would be faster if you used this suggest changes tool.